app/views/include/OpenAccessApisGroup.scala.html

```diff
@@ -14,47 +14,45 @@
  * limitations under the License.
  *@
 
-@import uk.gov.hmrc.thirdpartydeveloperfrontend.domain.models.views.SubscriptionRedirect
-@import views.helper.IdFormatter
 @import uk.gov.hmrc.thirdpartydeveloperfrontend.domain.models.applications.Application
 @import uk.gov.hmrc.apiplatform.modules.applications.domain.models.Collaborator
-@import uk.gov.hmrc.apiplatform.modules.apis.domain.models.ApiData
+@import uk.gov.hmrc.apiplatform.modules.apis.domain.models.ApiDefinition
 
 @this() 
 
 @(
   role: Collaborator.Role,
   app: Application,
-  openAccessApis: Map[ApiContext, ApiData]
+  openAccessApis: List[ApiDefinition]
 )(
   implicit messagesProvider: MessagesProvider,
   applicationConfig: ApplicationConfig,
   request: play.api.mvc.Request[Any]
 )
 
 @displayableContextName(apiContext: ApiContext) = @{apiContext.value.substring(apiContext.value.lastIndexOf('/')+1)}
-@displayableSectionId(apiContext: ApiContext, apiData: ApiData) = @{openAccessApis.toList.indexOf((apiContext, apiData))+1}
+@displayableSectionId(apiDefinition: ApiDefinition) = @{openAccessApis.indexOf(apiDefinition)+1}
 
 <div class="govuk-accordion" data-module="govuk-accordion" id="accordion-open-access">
-  @openAccessApis.toList.map { case (apiContext, apiData) =>
-    <div class="govuk-accordion__section" id="@displayableContextName(apiContext)-accordion-section">
+  @openAccessApis.map { apiDefinition =>
+    <div class="govuk-accordion__section" id="@displayableContextName(apiDefinition.context)-accordion-section">
       <div class="govuk-accordion__section-header">
         <h2 class="govuk-accordion__section-heading">
-          <span class="govuk-accordion__section-button" id="accordion-open-access-heading-@displayableSectionId(apiContext, apiData)">
-            @apiData.name
+          <span class="govuk-accordion__section-button" id="accordion-open-access-heading-@displayableSectionId(apiDefinition)">
+            @apiDefinition.name
           </span>
         </h2>
       </div>
 
-      <div id="accordion-open-access-content-@displayableSectionId(apiContext, apiData)" class="govuk-accordion__section-content" role="region" aria-labelledby="accordion-open-access-heading-@displayableSectionId(apiContext, apiData)">
-        @apiData.versions.toList.map { case (apiVersion, versionData) =>
-          @defining(s"${apiData.name} ${apiVersion.value} (${versionData.status.displayText})") { apiVersionFullName =>
+      <div id="accordion-open-access-content-@displayableSectionId(apiDefinition)" class="govuk-accordion__section-content" role="region" aria-labelledby="accordion-open-access-heading-@displayableSectionId(apiDefinition)">
+        @apiDefinition.versions.toList.map { case (apiVersionNbr, apiVersion) =>
+          @defining(s"${apiDefinition.name} ${apiVersionNbr.value} (${apiVersion.status.displayText})") { apiVersionFullName =>
```

Comment:
.value not needed when in string interpolation and the type has an explicit toString (versionnbr, context, appId etc do)

Reply:
You've opened a can of worms with this one :smile: